elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -62,20 +73,29 @@ $LOCAL-VARIABLES: (
 	}
 	&__window {
 		--theme: #{pfe-local(theme)};
-
+		//todo , this is a workaround:
```

Comment:
```suggestion
		// @TODO , this is a workaround
```
^ can you add a bit more information about what the workaround is and what it's accomplishing

Reply:
Opened an issue, updated comment to point to it:
https://github.com/patternfly/patternfly-elements/issues/1090